min_change_threshold: 0.01 # [rad]
max_speed: 0.5 # [rad/s]

analog_update_delay: 0.007 # [s]

zero_angle_signal: [850, 320, 512, 1509, 1500, 650]

analog_lower_signal_bound: 500
analog_upper_signal_bound: 2500

analog_lower_signal_bound_wrist: 600
analog_upper_signal_bound_wrist: 2400

digital_lower_signal_bound: 0
digital_upper_signal_bound: 1024

wrist_signal_zero_position: 1509
wrist_signal_90_degrees: 697

analog_speed: 2.0 # Change in signal per analog_update_delay
wrist_speed: 4.0 # Change in signal per analog_update_delay

publish_joint_states: true

interpolate_movement: true
interpolation_frequency: 10.0
